### ISSUE TITLE
Issue with the Variable inspector in VSCode's debug viewer for PowerShell

### ISSUE THREAD

**Issue by gitdram** [PowerShell/vscode-powershell#4604]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [ ] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

There is an issue in the Variable inspector in VSCode's debug viewer for PowerShell.
It appears that when an object has a custom format definition for displaying data, and if there is $null data for one of the object's default display properties, then the Variable inspector cannot display the data, but displays an error instead.
This is evident with the CosmosDb module, when using the Get-CosmosDbDocument cmdlet. If the Cosmos query uses a custom SELECT statement and does not return the _rid, _self, _attachments, and the _etag properties in the results, then the Variable inspector displays an error (see attachment).

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      5.1.19041.2673
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.2673
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Visual Studio Code Version

```console
1.78.2
b3e4e68a0bc097f0ae7907b217c1119af9e03435
x64
```

### Extension Version

```console
ms-vscode.powershell@2023.5.0
```

### Steps to Reproduce

It appears that when an object has a custom format definition for displaying data, and if there is $null data for one of the object's default display properties, then the Variable inspector cannot display the data, but displays an error instead.
This is evident with the CosmosDb module, when using the Get-CosmosDbDocument cmdlet. If the Cosmos query uses a custom SELECT statement and does not return the _rid, _self, _attachments, and the _etag properties in the results, then the Variable inspector displays an error (see attachment).

### Logs

[errors]

`Internal Error-System Management Automation Extended TypeSystemExceptionThe member_rid" is not present at System Management Automation PSAliasProperty LookupMember(String nameHashSet 1 visitedAliasesPSMemberinfo& returned Member
Boolean& hasCycle) at System Management Automation PSAliasProperty.LookupMember(String name) at System.Management Automation PSAliasProperty.get_Value()
-
] Internal Error
at Microsoft PowerShell EditorServices Services DebugAdapter.VariableDetails SafeGetValue(PSPropertyInfo psProperty)
at Microsoft PowerShell EditorServices Services DebugAdapter VariableDetails.<>c.<GetChildren>b_14_2(PSPropertyinfo at System.Linq.Enumerable WhereSelectEnumerablelterator 2 MoveNext(
p)
5] Internal Error -
at System.Collections.Generic.List 1.InsertRange(int32 index, IEnumerable 1 collection) at Microsoft.PowerShell.EditorServicesServices.DebugAdapter.VariableDetails.GetChildren(Object obj, ILogger logger) at Microsoft PowerShell.EditorServices Services DebugAdapter.VariableDetails....`

**Comment by JustinGrote** [comment]
Good catch! This is probably an edge case that hasn't been handled. Thanks for the detailed report.
